### DISCUSSION TITLE
Handle multipart/form-data parameters

### DISCUSSION THREAD

**Discussion by wolicodes** [grafana/grafana-json-datasource#255] [Ideas]
Those aren't currently supported afaik. Some APIs only handles this kind of parameters instead of classic JSON body or url params, but they don't seem usable with this plugin.

**Comment by marcusolsson** [comment]
Haven't tried it myself, but I don't see why it wouldn't work. The body isn't limited to JSON, you could put any day there, such as `value1=foo&value2=bar`. If you set the header to `Content-Type: multipart/form-data`, wouldn't that work?
